risks/MASVS-CRYPTO/2-weak-crypto-key-generation/weak-key-generation/android-weak-key-generation/demo-1/output.sarif

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Can you please remove the SARIF file for now? We can add later to keep the PR cleaner.